pkg/virtualKubelet/apiReflection/reflectors/outgoing/secrets.go

```diff
@@ -205,10 +224,9 @@ func (r *SecretsReflector) isAllowed(obj interface{}) bool {
 		klog.Error("cannot convert obj to secret")
 		return false
 	}
-	if sec.Type == corev1.SecretTypeServiceAccountToken {
-		return false
-	}
-	return true
+	// if this annotation is set, this secret will not be reflected to the remote cluster
+	_, ok = sec.Annotations["liqo.io/not-reflect"]
```

Comment:
can you assert also the annotation value? maybe "true" (just a proposal)?

Reply:
ok